go/libraries/doltcore/sqle/database.go

```diff
@@ -836,7 +858,7 @@ func (db Database) CreateIndexedTable(ctx *sql.Context, tableName string, sch sq
 	return db.createIndexedSqlTable(ctx, tableName, sch, idxDef, collation)
 }
 
-// Unlike the exported version CreateTable, createSqlTable doesn't enforce any table name checks.
+// Unlike the exported version CreateTable, createSqlTable doesn't enforce any table baseName checks.
```

Comment:
(minor) Might as well make this follow godoc format if you're going to put it in front of the method. Same comment for line 903.